sites/site-utilities/package.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "@microsoft/site-utilities",
+    "version": "0.0.0",
+    "description": "Utilities for the Creator and Component Explorer sites",
+    "sideEffects": false,
+    "private": true,
+    "scripts": {
+      "build": "tsc -p ./tsconfig.json",
+      "clean:dist": "node ../../build/clean.js dist",
+      "prepare": "yarn clean:dist && yarn build",
+      "prettier": "prettier --config ../../.prettierrc --write \"**/*.{ts,tsx}\"",
+      "prettier:diff": "prettier --config ../../.prettierrc \"**/*.{ts,tsx}\" --list-different",
+      "test": "yarn eslint && yarn build",
+      "eslint": "eslint . --ext .ts,.tsx",
+      "eslint:fix": "eslint . --ext .ts,.tsx --fix"
+    },
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "husky": {
```

Comment:
husky is not needed inside packages, it's only needed at root, per conversations with @chrisdholt .  Please remove configuration, including the dependency.